presto-spi/src/main/java/com/facebook/presto/spi/statistics/StatisticsValue.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+public final class StatisticsValue
```

Comment:
In any case, I would rename it to `ValueEstimate`. Keep in mind that this is estimate for singular values, e.g: number of rows. When we have `min, max` ranges, then we should also have `RangeEstimate`, which covers entire population with confidence `X`, e.g. 95%